pytorch_blade/src/ltc/disc_compiler/disc_compiler.cpp

```diff
@@ -11,18 +11,18 @@
 
 #include "ltc/disc_compiler/disc_compiler.h"
 
-#include "common_utils/utils.h"
-#include "ltc/disc_compiler/passes/disc_fuser.h"
-#include "ltc/disc_compiler/passes/register_disc_class.h"
-
 #include <ATen/Functions.h>
+#include <torch/csrc/jit/jit_log.h>
 #include <torch/csrc/jit/passes/dead_code_elimination.h>
 #include <torch/csrc/jit/passes/shape_analysis.h>
 #include <torch/csrc/lazy/backend/backend_data.h>
 #include <torch/csrc/lazy/backend/backend_device.h>
 #include <torch/csrc/lazy/ts_backend/ts_backend_impl.h>
 #include <torch/csrc/lazy/ts_backend/ts_lowering_context.h>
 #include <torch/script.h>
+#include "common_utils/utils.h"
```

Comment:
Do we have a linter for include order?

Reply:
Yes, we use clang-format : https://github.com/alibaba/BladeDISC/blob/main/.pre-commit-config.yaml#L12